Drivers/imu/src/icm42688.cpp

```diff
@@ -0,0 +1,137 @@
+// icm42688.cpp
+
+#include "icm42688.hpp"
+#include "stm32l5xx_hal_conf"
```

Comment:
this line looks weird, it doesn't have a file extension

Reply:
Are you on Linux? Try building on linux, that *might* tell you where the error is.

Reply:
I also just went through the build - I can't find any reference to "icm" or "hal_conf", are we sure your files are included properly in cmakelists.txt and are being built?